brave/brave-core
# Fetch accounts from Ledger device

First step of https://github.com/brave/brave-browser/issues/17971 

- Added API proxy to provide ledgerjs Hardware keyring
- Fetch accounts from Ledger device

Saving accounts will be implemented in separate PR

![image](https://user-images.githubusercontent.com/2965009/133224141-c7fbf040-0066-4bc2-a9ec-b8c96c8613c0.png)

![image](https://user-images.githubusercontent.com/2965009/133269389-44aa4327-e69e-4a8b-8c33-72054fcf6fe7.png)

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Open Add Hardware Account section in Wallet
- Connect ledger, try to click connect, the dialog should show error
- Open an app in Ledger device, try to click connect again, it shows available accounts to check
